helm-charts/interoperator/values.yaml

```diff
@@ -54,6 +54,21 @@ interoperator:
     provisionerWorkerCount: 2
     schedulerType: label-selector
 
+operator_apis:
+  port: 9297
+  username: admin
+  password: secret
+  host: op.ingress.sf21-intop.interop.shoot.canary.k8s-hana.ondemand.com
+  image:
+    repository: servicefabrikjenkins/operatorapis
+    tag: 0.1.0
+    pullPolicy: Always
+  service:
+    type: LoadBalancer
```

Comment:
Might be better to default to ClutserIP type

Reply:
done